### DISCUSSION TITLE
feat: Function Calling Under OpenAI Format

### DISCUSSION THREAD

**Discussion by anhphong22** [langfuse/langfuse#1633] [Ideas]
### Describe the feature or potential improvement

To return the output as desired, I request a new feature the same as `function calling` with the input and UI as the image below:

![image](https://github.com/langfuse/langfuse/assets/74782468/f0918084-5240-49cb-b5f3-87134a674bad)

### Additional information
**UI: Playground** 

_No response_

**Comment by marcklingen** [comment]
Tool calling is supported by Langfuse Tracing. What are you missing?

**Reply by anhphong22** [reply]
But I want to ingrate this tool as the feature on UI of prompt. This helps prompt engineers to boost their performance.

**Reply by anhphong22** [reply]
![image](https://github.com/langfuse/langfuse/assets/74782468/9ab57994-19a9-4585-be06-4063638a92ce)

**Reply by anhphong22** [reply]
@marcklingen I spoil this feature from Lunary.ai

**Reply by marcklingen** [reply]
Is this related to a prompt playground?

**Reply by anhphong22** [reply]
Yes, you can pass function call in UI check [here](https://platform.openai.com/docs/guides/function-calling)
